unleash-incognia.gemspec

```diff
@@ -0,0 +1,23 @@
+require_relative "lib/unleash/incognia/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "unleash-incognia"
```

Comment:
I wondering if it should be `unleash-incognia-strategy` or something like that.
What do you think?

Reply:
To do so, the structure should be `Unleash::Incognia::Strategy`. Also, my intention here is to have a more generic gem so we can add more stuff here, if we like. wdyt?

Reply:
I get it :+1: